_test/test_algorithms/test_cut.m

```diff
@@ -302,23 +302,6 @@ function test_you_can_take_a_cut_with_nopix_arg_and_output_to_file(obj)
             
             assertEqualToTol(output_obj, ref_object, 'ignore_str', true);
         end
-        function test_warning_issued_if_bin_size_exceeds_page_size(obj)
```

Comment:
because the issue was fixed and no problem with large bin size exist.  A routine splits large bin into pages and deals with it page by page. Its routine operation now and bunch of unit tests has been added to confirm its operability